src/main/java/org/broadinstitute/dsde/consent/ontology/resources/ErrorResource.java

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.consent.ontology.resources;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import org.broadinstitute.dsde.consent.ontology.model.Error;
+import org.eclipse.jetty.server.Request;
+
+@Path("error")
+public class ErrorResource {
+
+  @GET
+  @Path("404")
+  @Produces("application/json")
+  public Response notFound(@Context HttpServletRequest request) {
+    String originalUri = ((Request) request).getOriginalURI();
+    String msg = String.format("Unable to find requested path: '%s'", originalUri);
```

Comment:
There must be something weird happening with this unicode decode: 
<img width="637" alt="image" src="https://user-images.githubusercontent.com/17440010/211620565-0fc2a0b6-87a3-483d-beb8-1f58b3762a5b.png">

Reply:
That's url encoding happening inside `getOriginalURI` ... gonna see if I can make that prettier.